ansible/roles/setup_course/templates/env.setup_course.j2

```diff
@@ -9,8 +9,11 @@ ILLUMIDESK_DIR={{working_dir}}
 # api token should be equal to value set for jupyterhub
 JUPYTERHUB_API_TOKEN={{jhub_api_token}}
 
+# external facing api url
+JUPYTERHUB_BASE_URL={{base_url}}
```

Comment:
I think we can avoid the use of this variable and only use the base_url (from ansible). There are three files using this variable now so it can be confuse

Reply:
I left it in here with the idea that once we refactor the env.* files it would make more sense to leave the base url env var intact so that the course object can fetch the same value used by the hub.